packages/rest/src/__tests__/integration/request-context.integration.ts

```diff
@@ -141,5 +157,9 @@ function contextObservingHandler(
   _sequence: DefaultSequence,
 ) {
   observedCtx = ctx;
+  // Add a subscriber to verify `close()`
+  ctx.subscribe(() => {});
```

Comment:
@raymondfeng 
Just wondering how an empty function can be passed when the function type is:
```
export type ContextObserverFn = (
  eventType: ContextEventType,
  binding: Readonly<Binding<unknown>>,
  context: Context,
) => ValueOrPromise<void>;

```
If you could clarify. Thanks

Reply:
The empty function is compatible with `ContextObserverFn` as it does not care about the args and just return `void`.